### ISSUE TITLE
http 429 error 

### ISSUE THREAD

**Issue by berkay1337x** [HerrEurobeat/steam-comment-service-bot#65]
Hello when i start the bot after a few minutes it start say http 429 error.I added 19 account with no 5 dolar limit. How can i fix this error help me please.

**Comment by HerrEurobeat** [comment]
Hey, thanks for opening an issue.
This isn't a bug and you can't "fix" this since it is a correctly working cooldown feature by Steam. I even have a description in my Wiki on this error: https://github.com/HerrEurobeat/steam-comment-service-bot/wiki/Errors,-FAQ-&-Common-problems#steam-errors  

Basically every server will apply a cooldown on an IP if they make too many requests in specific time period.  
You can only try to get these cooldown "errors" less quickly by increasing the commentdelay and the globalcommentcooldown or just in general by commenting less often, less comments at one time or by using proxies to use multiple IPs. (Although I heard some proxies don't work with Steam)

The only thing you can do is to make a less requests/comments in a time frame or use less accounts on one IP.  

The "error" is basically a feature that every service has so that people can't spam their servers.

**Comment by berkay1337x** [comment]
I dont understand how can i add proxy tell me please.

**Comment by HerrEurobeat** [comment]
Your bot folder should contain a proxies.txt file that got created by the bot the first time you started it.  
You can add proxies to it by putting a HTTP Proxy in each line. The accounts will then be evenly split to use your native IP and all available proxies by the bot.  

The proxies must be HTTP proxies and should have the format of `IP:Port` (example: `113.117.10.236:4216`). I personally have no experience of which proxy-selling-services work with Steam but I heard that quite a few are blocked by Steam so you have to do a quick research which proxy services work with Steam!  

If you have another question feel free to ask!  
I personally am not using any proxies with my 5 accounts (limited and friend with the recieving profile) and I have quite good success. (commentdelay: 20000 and 5-10 comments each 1-2 hours)

**Comment by berkay1337x** [comment]
I wanna sent comment bot to my review or artworks how can i do this?

**Comment by HerrEurobeat** [comment]
Sorry but the bot only supports profile comments.

**Comment by HerrEurobeat** [comment]
I hope this solved your question. If it didn't feel free to reopen this issue.
If you've got any other question feel free to open another issue!
Have a nice day!
